compiler/pipes/register-kphp-configuration.cpp

```diff
@@ -19,7 +19,7 @@ bool RegisterKphpConfiguration::check_function(FunctionPtr function) const {
 void RegisterKphpConfiguration::on_start() {
   const auto klass = current_function->class_id;
   kphp_error(
-    !klass->members.has_any_instance_method() &&
+//    !klass->members.has_any_instance_method() &&
```

Comment:
Due to the transfer of processing `KphpConfiguration`, an unfiltered constructor remains in it and because of this, the check here is true and therefore an error is produced.
The line will be uncommented when the color definition goes to the `json` file.